install.sh

```diff
@@ -1,25 +1,51 @@
+#!/bin/bash
+
+###############################################################################
+# Instalation Script for Gitshots
+# Repository: https://github.com/ranman/gitshots-client
+# gitshots.com - Take a picture and collect some stats every time you commit!
+###############################################################################
+
+USERHOME=$(dscl . -read /Users/`whoami` NFSHomeDirectory | awk -F": " '{print $2}')
```

Comment:
In the beginning I was using $HOME, unfortunately that does not work for homebrew its home is not the user home but something like: /private/tmp/.brew/gitshots-client20150923-16814-1l3nitf/